app/assets/javascripts/utils/article_utils.js

```diff
@@ -1,19 +1,23 @@
 export default class ArticleUtils {
-// Returns article or item, based on home Wiki project.
+  // Returns article or item, based on home Wiki project.
   static articlesOrItems(project) {
     return project === 'wikidata' ? 'items' : 'articles';
   }
 
-// Returns article or item message, based on articles or items keyword.
+  // Returns article or item message, based on articles or items keyword.
   static articlesOrItemsMsg(messageKey, articlesOrItems, defaultarticlesOrItems = 'articles') {
     return I18n.t(`${articlesOrItems}.${messageKey}`, {
       defaults: [{ scope: `${defaultarticlesOrItems}.${messageKey}` }]
     });
   }
 
-// Returns article or item message, based on home Wiki project.
-static I18n(messageKey, project) {
-  const articlesOrItems = ArticleUtils.articlesOrItems(project);
-  return ArticleUtils.articlesOrItemsMsg(messageKey, articlesOrItems);
-}
+  // Returns article or item message, based on home Wiki project.
+  static I18n(messageKey, project) {
+    const articlesOrItems = ArticleUtils.articlesOrItems(project);
+    return ArticleUtils.articlesOrItemsMsg(messageKey, articlesOrItems);
+  }
+
+  static chooseMsg(project, msg) {
```

Comment:
Everything in this PR looks good to me now, except for the name of this function. Can you think of a name that more clearly communicates its intended use of possibly appending a project-specific suffix to a message key? (I think that, while the current implementation only knows about Wikidata, we could imagine the behavior changing later to add a different suffix for a different project.)

Reply:
appendProjectSufix? But for Wikipedia nothing is appending...

Reply:
Maybe just `projectSuffix`? That gets across the idea of varying the string based on the project, without implying that one will always be added.

Reply:
Done!